lib/grplatt.gi

```diff
@@ -3187,21 +3187,22 @@ local act,offset,G,lim,cond,dosub,all,m,i,j,new,old;
   return all;
 end);
 
-#############################################################################
-##
-#F  ContainedConjugates( <G>, <A>, <B> )
-##
-InstallMethod(ContainedConjugates,"finite groups",IsFamFamFam,[IsGroup,IsGroup,IsGroup],0,
-function(G,A,B)
-local l,N,dc,gens,i;
+DoContainedConjugates:=function(arg)
+local G,A,B,onlyone,l,N,dc,gens,i;
+  G:=arg[1];
+  A:=arg[2];
+  B:=arg[3];
+  if Length(arg)>3 then onlyone:=arg[4]; else onlyone:=false;fi;
```

Comment:
Could also do this:
```suggestion
DoContainedConjugates:=function(G,A,B,onlyone...)
local l,N,dc,gens,i;
  if Length(onlyone)>0 then onlyone:=onlyone[1]; else onlyone:=false;fi;
```

Reply:
Or this
```suggestion
DoContainedConjugates:=function(G,A,B,onlyone)
local l,N,dc,gens,i;
```
and then change one InstallMethod below to use the following function instead: `{G,A,B}->DoContainedConjugates(G,A,B,false)`